Scripts/LookupCSV/LookupCSV.yml

```diff
@@ -0,0 +1,31 @@
+name: LookupCSV
+script: '-'
+type: python
+tags:
+  - file
+  - csv
+  - Utility
+comment: Parses a CSV and looks for a specific value in a specific column, returning
+  a dict of the entire matching row.
+enabled: true
+args:
+  - name: entryID
+    required: true
+    description: EntryID of CSV file.
+  - name: header_row
+    description: CSV file has a header row
+  - name: column
+    description: Column to search for value in
+  - name: value
```

Comment:
please make this required, as without it there is nothing to search for

Reply:
If you don't specify column, it simply extracts the entire CSV into json format and still works. Unfortunately (as far as I know) there's no way to have demisto args be dependent on each other so I can't make value be required.

Reply:
But then the script doesn't exactly do what it was meant to, I think that we should not return a result for this, or at least mention that no results were found(Both in the War room and in the context)